src/components/SubscribeButton/SubscribeButton.tsx

```diff
@@ -40,11 +32,18 @@ const _SubscribeButton = ({
         }
       >
         <Button
-          className={twMerge('p-0', className)}
+          className={twMerge(
+            'flex items-center justify-center gap-3 p-0',
```

Comment:
I think the `p-0` here might be a bit squishy when the button is in active/loading state:
![Screenshot 2023-04-05 at 10.23.17 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/w3rL99SY56BQpB9YUooj/ebcfa736-1386-4474-8dde-e575e98316eb/Screenshot%202023-04-05%20at%2010.23.17%20AM.png)

Reply:
agree, but this is antd at work unfortunately. We'd have to rebuild the button I think to get it behaving right

Reply:
can't we just nuke p-0?

Reply:
done